lib/src/components/NxTabs/stateful/NxStatefulTabs.tsx

```diff
@@ -30,24 +30,22 @@ const NxStatefulTabs = function NxStatefulTabsElement(props: Props) {
     return null;
   }
 
-  const firstTabId = (Children.toArray(originalTabList.props.children)[0].props as NxTabProps).id;
-  const [activeTabId, setActiveTabId] = useState(firstTabId);
+  const firstActiveTab = (Children.toArray(originalTabList.props.children)[0].props as NxTabProps).id;
+  const [activeTab, setActiveTab] = useState(firstActiveTab);
 
   const tabList = cloneElement(originalTabList, {
     children: Children.map(originalTabList.props.children, (child) => cloneElement(child, {
```

Comment:
I would have expected `NxTabs` to use a pattern similar to this to consolidate activation clicks into a single `onChange` handler or similar, as I did on my branch.

Reply:
db4ffe8